### DISCUSSION TITLE
Make steps consecutive

### DISCUSSION THREAD

**Discussion by bdvorianov** [Escape-Technologies/mookme#78] [Q&A] [answered]
When I trigger some of hooks that have more than one step, they all run at the same time. Is there a way to make the next script run after the previous one was successful?

P.S. Now, when I've tried it, it seem more useful for me to have it as it is, so I know all the issues at the time, but still curious.

**Comment by LMaxence** [comment, marked as answer]
Hello !

There is a `serial` option available in the [step definition](https://mookme.org/references/#available-options). If you mark a step as "serial", every following step will await for this very step to be terminated before it starts

**Reply by bdvorianov** [reply]
Thank you for such a quick reply. 🙏🏻
